src/DataFixtures/ClientFixtures.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * (c) Adrien PIERRARD
+ */
+
+namespace App\DataFixtures;
+
+use App\Entity\Client;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+/**
+ * Class ClientFixtures.
+ */
+class ClientFixtures extends Fixture
+{
+    /**
+     * A prefix reference constant for Client.
+     *
+     * @var string
+     */
+    public const CLIENT_REFERENCE_PREFIX = 'client_';
+
+    /**
+     * Load fixtures in Client table.
+     *
+     * @param ObjectManager $manager
+     *
+     * @return void
+     */
+    public function load(ObjectManager $manager): void
+    {
+        $client = new Client();
+        $client
+            ->setName('Bubble PhoneShop')
+            ->setEmail('contact@bubble.com')
+        ;
+        $manager->persist($client);
+        $this->addReference(self::CLIENT_REFERENCE_PREFIX.'1', $client);
+
+        $client = new Client();
+        $client
+            ->setName('PandaKing Smartphone')
+            ->setEmail('contact@lovely-panda.fr')
+        ;
+        $manager->persist($client);
+        $this->addReference(self::CLIENT_REFERENCE_PREFIX.'2', $client);
+
+        $client = new Client();
+        $client
+            ->setName('PhonePhone Palace')
+            ->setEmail('contact@phonephone.call')
+        ;
+        $manager->persist($client);
+        $this->addReference(self::CLIENT_REFERENCE_PREFIX.'3', $client);
+
+        $manager->flush();
+    }
```

Comment:
It's important to code the fixtures like you would code other classes.
It will be code to maintain.
Here too much copy/paste.
```suggestion

    private $samples = [
        ['Bubble PhoneShop', 'contact@bubble.com'],
        ['PandaKing Smartphone', 'contact@lovely-panda.fr'],
        ['PhonePhone Palace', 'contact@phonephone.call'],
    ];

    /**
     * Load fixtures in Client table.
     *
     * @param ObjectManager $manager
     *
     * @return void
     */
    public function load(ObjectManager $manager): void
    {
        foreach ($this->samples as $k => $sample) {
            $client = new Client();
            $client
                ->setName($sample[0])
                ->setEmail($sample[1]);
            $manager->persist($client);
            $this->addReference(self::CLIENT_REFERENCE_PREFIX . ($k + 1), $client);
        }

        $manager->flush();
    }
```